spa/src/components/pageEditor/editInterface/pageSetup/PublishWidget.js

```diff
@@ -31,6 +31,15 @@ function PublishWidget({ publishDate, onChange, errors }) {
     onChange(new Date());
   };
 
+  const hasValidPaymentProvider = paymentProvider && paymentProvider.stripe_verified ? true : false;
```

Comment:
Think this can just be

```suggestion
  const hasValidPaymentProvider = paymentProvider && paymentProvider.stripe_verified;
```